test/test.js

```diff
@@ -1,46 +1,50 @@
 'use strict';
 
-var expect = require('chai').expect;
-var numFormatter = require('../index');
+const expect = require('chai').expect;
```

Comment:
is it better to make chai a constant ?

Reply:
The code in red was the practise/demo code. the test file only has consts or lets.